mflowgen/Tile_PE/constraints/constraints.tcl

```diff
@@ -43,7 +43,8 @@ set_constraint_mode default_c
 set_false_path -from [get_ports config* -filter direction==in] -to [get_ports SB* -filter direction==out]
 
 # Paths from config input ports to SB registers
-set sb_reg_path SB_ID0_5TRACKS_B*_PE/REG_T*_B*/value__CE/value_reg[*]/*
+set sb_reg_path SB_ID0_5TRACKS_B*_PE/REG_T*_B*/value__CE/*
```

Comment:
Need to update these constraints so we don't get massive ULVT utilization.

Reply:
@norabarlow Any idea what it needs to become?

Reply:
Sadly no. I'm not sure why this doesn't work...

Reply:
Fixed with most recent commit.